zeebe/gateway-rest/src/main/java/io/camunda/zeebe/gateway/rest/SearchQueryResponseMapper.java

```diff
@@ -932,8 +932,8 @@ private static List<DecisionInstanceResult> toDecisionInstances(
 
   public static DecisionInstanceResult toDecisionInstance(final DecisionInstanceEntity entity) {
     return new DecisionInstanceResult()
-        .decisionInstanceKey(KeyUtil.keyToString(entity.decisionInstanceKey()))
-        .decisionInstanceId(entity.decisionInstanceId())
+        .decisionEvaluationKey(KeyUtil.keyToString(entity.decisionInstanceKey()))
+        .decisionEvaluationInstanceKey(entity.decisionInstanceId())
```

Comment:
❓ here is the only place where we map from the legacy "ids/keys" to the new naming schema? Thus I would recommend we leave a comment here. wdyt?

Reply:
That we decided to rename this on the "UI" level, but under the hood it's still stored in the old fields and the next engineer should not mistake the naming.